tests/components/litejet/test_light.py

```diff
@@ -50,8 +49,10 @@ def on_load_deactivated(number, callback):
         self.mock_lj.on_load_deactivated.side_effect = on_load_deactivated
 
         assert setup.setup_component(
-            self.hass, litejet.DOMAIN, {"litejet": {"port": "/tmp/this_will_be_mocked"}}
-        )
+            self.hass, litejet.DOMAIN,
+            {"litejet": {
+                "port": "/tmp/this_will_be_mocked"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Probable insecure usage of temp file/directory.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4620096)